src/Lykke.Service.BlockchainWallets.Services/AddressService.cs

```diff
@@ -43,7 +46,12 @@ public async Task<string> MergeAsync(string blockchainType, string baseAddress,
 
             if (!string.IsNullOrEmpty(addressExtension))
             {
-                if (!await _capabilitiesService.IsPublicAddressExtensionRequiredAsync(blockchainType))
+                var capabilityQueryResult = _blockchainExtensionsService.IsPublicAddressExtensionRequired(blockchainType);
+                if (!capabilityQueryResult.HasValue)
+                {
```

Comment:
What's happen inside this "if" statement? looks like nothing

Reply:
I missed a `throw` clause for exception here. Added it now.